### PR TITLE
Dataclasses Added

### PR DESCRIPTION
**Pull request by roshanborkar01**

### CODE REVIEW

**Comment by roshanborkar01** [comment]
Dataclasses added for the rest `processing completed rejected and error`.

File: Server/sprawl_calculator/wup_calculator/data_definitions.py [hunk 1]
```diff
@@ -5,3 +5,23 @@
...
+
+@dataclass
+class WUPCalculationRequestProcessing:
+	status: str
+	processing_id: str
+
+@dataclass
+class WUPCalculationRequestCompleted:
+	status: str
+	processing_id: str
```
Comment (hrishiballal) [inline Server/sprawl_calculator/wup_calculator/data_definitions.py:26]: There is a extra result field here, that needs to be added: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L271

File: Server/sprawl_calculator/wup_calculator/data_definitions.py [hunk 2]
```diff
@@ -5,3 +5,23 @@
 class WUPCalculationRequestActivating:
 	status: str
 	processing_id: str
+
+@dataclass
+class WUPCalculationRequestProcessing:
+	status: str
+	processing_id: str
```
Comment (hrishiballal) [inline Server/sprawl_calculator/wup_calculator/data_definitions.py:19]: In addition, I just added created at and updated at to the documentation: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L286 , these should be added to the definitions as well.

**Review by hrishiballal — CHANGES_REQUESTED** [review]

File: Server/sprawl_calculator/wup_calculator/data_definitions.py (hunk 2 above)
Comment (roshanborkar01) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:19]: Ok, I will check and update same and pr for review.

Comment (roshanborkar01) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:19]: New fields are added to the dataclasses.

File: Server/sprawl_calculator/wup_calculator/data_definitions.py (hunk 1 above)
Comment (hrishiballal) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:26]: The results field needs to be updated, see the WUPCalculationResult definition here: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L284

**Review by hrishiballal — CHANGES_REQUESTED** [review]

File: Server/sprawl_calculator/wup_calculator/data_definitions.py (hunk 1 above)
Comment (roshanborkar01) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:26]: Type for field result is changed to WUPCalculationResult as per documentation.  do we need to create or import new dataclass for WUPCalculationResult ?

Comment (hrishiballal) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:26]: Yes, lets create a new dataclass called WUPCalculationResult per definition here: https://github.com/geodesignhub/urban-sprawl-calculation-service/blob/master/api/sprawl-calculation-service.yaml#L202 and then there should be a field here in WUPCalculationRequestCompleted called result of type WUPCalculationResult `result: WUPCalculationResult`

**Review by hrishiballal — CHANGES_REQUESTED** [review]

**Comment by roshanborkar01** [comment]
Thanks for the review.I will do that and Let u know.

File: Server/sprawl_calculator/wup_calculator/data_definitions.py (hunk 1 above)
Comment (hrishiballal) [reply Server/sprawl_calculator/wup_calculator/data_definitions.py:26]: The result should be added to the successful processing class

**Pull request merged**